Source/Core/Core/State.cpp

```diff
@@ -71,7 +71,7 @@ static Common::Event g_compressAndDumpStateSyncEvent;
 static std::thread g_save_thread;
 
 // Don't forget to increase this after doing changes on the savestate system
-static const u32 STATE_VERSION = 73;  // Last changed in PR 4651
+static const u32 STATE_VERSION = 74;  // Last changed in PR 3701
```

Comment:
Fixed. Though it doesn't really matter, because PR 3701 isn't going to get merged and introduce a different savestate layout.